thirdparty/cmake_modules/koreader_thirdparty_git.cmake

```diff
@@ -69,79 +60,76 @@ if(should_clone)
   if(error_code)
     message(FATAL_ERROR \"Failed to clone repository: '${git_repository}'\")
   endif()
+  execute_process(
+    COMMAND \"${git_EXECUTABLE}\" -C \"${clone_checkout}\" sparse-checkout init
```

Comment:
This isn't supported yet. But thanks for providing the update.

Reply:
Note: the code should be robust to `sparse-checkout` not being supported (exit codes are ignored for that reason).